src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -0,0 +1,2862 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Tree Root="IOperation">
+
+
+  <!--
+      To regenerate the operation nodes, run eng/generate-compiler-code.cmd
+
+      The operations in this file are _ordered_! If you change the order in here,
+      you will affect the ordering in the generated OperationKind enum, which will
+      break the public api analyzer. All new types should be put at the end of this
+      file in order to ensure that the kind does not continue to change.
+
+      UnusedOperationKinds indicates kinds that are currently skipped by the OperationKind
+      enum. They can be used by future nodes by inserting those nodes at the correct point
+      in the list.
+        -->
+
+  <UnusedOperationKinds>
+    <Entry Value="0x1D"/>
+    <Entry Value="0x62"/>
+    <Entry Value="0x64"/>
+  </UnusedOperationKinds>
```

Comment:
what does this mean?

Reply:
These were skipped for various reasons, usually due to changing apis that meant something wasn't shipped. I had to skip them in the OperationKind enum though, so they're enumerated here, and future operations could make use of these kinds.